docs/code-reviews.md

```diff
@@ -1,4 +1,67 @@
-!!! note "These documents are a work-in-progress"
-    This section needs to be written! A "code review" is an important quality assurance practice. Every time you make a Pull Request in Github, you should ask a collaborator to do a review. [This video](https://github.com/features/code-review/) is a good starting point.
+## What is a code review?
 
----8<-- 'includes/glossary.md'
+A code review is a critical quality assurance practice. It involves systematically
+checking a fellow programmer's (or researcher's) code for errors or possible 
+improvements. 
+
+It is an everyday activity in the software development world and improves
+both quality and speed in programming. 
+
+## How to ask for a review
+
+Github has a feature built into the workflow that allows easy code review with collaborators, and we would recommend 
+that you use this tool. 
+
+![image](https://user-images.githubusercontent.com/25401512/109499428-cdb2f680-7a8c-11eb-9d77-4e53f7f1c57c.png)
```

Comment:
I know this is a PITA but can we include the images in the documentation repo? You can see an example here https://github.com/ebmdatalab/datalab-team-manual/blob/main/docs/calendars.md

Reply:
the images don't render when i look at this link for some reason. can you see them>?

Reply:
they render correctly on https://team-manual.thedatalab.org/calendars/ , but not in the github view because they're relative links :/

Reply:
So I still have a preference for including all the assets in the repo, i.e. copying images to an image folder and referencing as relative links.

That way the entire documentation is self-contained and could, e.g., be hosted offline, etc.

Reply:
moved into the images folder. will push shortly